kinds/structured/playlist/playlist_kind.cue

```diff
@@ -0,0 +1,47 @@
+package kind
+
+name: "Playlist"
+maturity: "committed"
+
+lineage: seqs: [
+	{
+		schemas: [
```

Comment:
are we *moving* the cue files here?  or is this another copy?  -- presumably the coremodel one goes away

Reply:
yes, they'll move, `pkg/coremodel` will be removed, and we will never see that damn word again 🎉